modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -0,0 +1,316 @@
+          when APPOINTMENT_TYPES[:va_video_connect_atlas],
+            APPOINTMENT_TYPES[:va_video_connect_home],
+            APPOINTMENT_TYPES[:va_video_connect_gfe]
+            if telehealth
+              address = telehealth.dig(:atlas, :address)
+
+              if address
+                location[:address] = {
+                  street: address[:street_address],
+                  city: address[:city],
+                  state: address[:state],
+                  zip_code: address[:zip_code],
+                  country: address[:country]
+                }
+              end
+
+              location[:url] = telehealth[:url]
+              location[:code] = telehealth.dig(:atlas, :confirmation_code)
+            end
+          else
+            location[:id] = appointment_hash.dig(:location, :id)
+            location[:name] = appointment_hash.dig(:location, :name)
+            address = appointment_hash.dig(:location, :physical_address)
+            if address.present?
+              location[:address] = {
+                street: address[:line]&.join(' ')&.strip,
+                city: address[:city],
+                state: address[:state],
+                zip_code: address[:postal_code]
+              }
+            end
+            location[:lat] = appointment_hash.dig(:location, :lat)
+            location[:long] = appointment_hash.dig(:location, :long)
+            location[:phone] = location_phone(appointment_hash)
+          end
+
+          location
+        end
+        # rubocop:enable Metrics/MethodLength
+
+        def location_phone(appointment_hash)
+          phone = appointment_hash.dig(:location, :phone, :main)
+          return nil unless phone
+
+          # captures area code (\d{3}) number (\d{3}-\d{4})
+          # and optional extension (until the end of the string) (?:\sx(\d*))?$
+          phone_captures = phone.match(/^(\d{3})-(\d{3}-\d{4})(?:\sx(\d*))?$/)
+
+          if phone_captures.nil?
+            Rails.logger.warn(
+              'mobile appointments failed to parse VAOS V2 facility phone number',
+              facility_id: appointment_hash.dig(:location, :id),
+              facility_phone: phone
+            )
+            return nil
+          end
+
+          {
+            area_code: phone_captures[1].presence,
+            number: phone_captures[2].presence,
+            extension: phone_captures[3].presence
+          }
+        end
+
+        def time_zone(facility_id)
+          facility = Mobile::VA_FACILITIES_BY_ID["dfn-#{facility_id}"]
+          facility ? facility[:time_zone] : nil
+        end
+
+        def healthcare_provider(practitioners)
+          return nil if practitioners.nil? || practitioners.none? { |prac| prac[:name] }
+
+          practitioners.map do |practitioner|
+            first_name = practitioner.dig(:name, :given)&.join(' ')&.strip
+            last_name = practitioner.dig(:name, :family)
+            [first_name, last_name].compact.join(', ').presence
```

Comment:
no, previously we just put a space between them. I was just worried about the `given` array of strings instead of first name. but it probably is just first and middle name and can still be space deliminated, that way I can keep it all on one line and deliminate separate practitioners by commas